sphinx/source/docs/user_guide/interaction.rst

```diff
@@ -48,16 +48,58 @@ Now you have learned how to link brushing between plots.
 Adding Widgets
 --------------
 
+Widgets are interactive controls that can be added to Bokeh applications to 
+provide a front end user interface to a visualization. They can drive new computations, 
+update plots, and connect to other programmatic functionality. When used with the 
+Bokeh server, widgets can run arbitrary Python code, enabling complex applications.
+Widgets can also be used without the Bokeh server in static HTML documents through the 
+browser's Javascript runtime.
+
+To use widgets, you must add them to your document and define their functionality. 
+Widgets can be added directly to the document root or nested inside a layout. There
+are two ways to program a widget's functionality:
+
+    * Use the ``CustomJS`` callback (see below). This will work in static HTML documents.
```

Comment:
We've been favoring "standalone" over "static", since "static" is easy to confuse with "no interactions"